conda/recipe/build.sh

```diff
@@ -59,11 +59,17 @@ cmake -DCUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda \
       -DCMAKE_INSTALL_PREFIX=$PREFIX                      \
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-Release}     \
       -DBUILD_TEST=${BUILD_TEST:-ON}                      \
+      -DBUILD_DALI_KERNELS=${BUILD_DALI_KERNELS:-ON}      \
+      -DBUILD_DALI_PIPELINE=${BUILD_DALI_PIPELINE:-ON}    \
+      -DBUILD_DALI_OPERATORS=${BUILD_DALI_OPERATORS:-ON}  \
       -DBUILD_BENCHMARK=${BUILD_BENCHMARK:-ON}            \
       -DBUILD_NVTX=${BUILD_NVTX}                          \
       -DBUILD_PYTHON=${BUILD_PYTHON:-ON}                  \
       -DBUILD_LMDB=${BUILD_LMDB:-ON}                      \
       -DBUILD_JPEG_TURBO=${BUILD_JPEG_TURBO:-ON}          \
+      -DBUILD_FFMPEG=${BUILD_FFMPEG:-ON}                  \
```

Comment:
remove, BUILD_FFMPEG is an internal variable (BUILD_NVDEC is the exposed option)

Reply:
Done